sdk/src/wallet/account/operations/transaction/prepare_output.rs

```diff
@@ -82,6 +76,13 @@ where
             if let Some(sender) = features.sender {
                 first_output_builder = first_output_builder.add_feature(SenderFeature::new(sender))
             }
+
+            if let Some(issuer) = features.issuer {
+                if let OutputBuilder::Basic(_) = first_output_builder {
+                    return Err(crate::wallet::Error::MissingParameter("nft_id"));
+                }
+                first_output_builder = first_output_builder.add_immutable_feature(IssuerFeature::new(issuer));
+            }
         }
 
         if let Some(unlocks) = params.unlocks {
```

Comment:
for the unlocks, do we want to check a time actually?

Reply:
Check the time for what?

Reply:
sensible times? Ideally not before the latest milestone timestamp. but as this will change again soon, 🤷‍♂️

Reply:
Well before the latest timestamp would also do no harm and protocol allows it